pkg/apis/v1alpha1/types.go

```diff
@@ -154,3 +154,54 @@ type ThanosQuerierSpec struct {
 // It should always be reconstructable from the state of the cluster and/or outside world.
 type ThanosQuerierStatus struct {
 }
+
+// MonitoringStackConfig configures the deployment of the different operators managed
```

Comment:
@sthaha @jan--f Here's a preliminary preview of the knobs exposed by the new config CRD please let me know what do you think